packages/phone-number-privacy/signer/src/domain/domain.service.ts

```diff
@@ -139,6 +140,11 @@ export class DomainService implements IDomainService {
       hash: domainHash(domain),
     })
 
+    const key: Key = {
+      type: KeyType.domains,
+      version: 1,
```

Comment:
Just realized that this is a challenge because we need to be able to have the combiner decide which key the request uses, if the client does not specify. My inclination is to either A) have the request be placed in a `body.request` field, and add a `body.combiner` field to allow for the combiner to transmit metadata such as this or B) add a custom http header. B is slightly less elegant, but it does have the advantage of working for both the phone number privacy and the domains API.

Reply:
Maybe you can think of a better solution.

Reply:
I ended up going with option B. Let me know what you think